test-boot1-compatibility/README.md

```diff
@@ -0,0 +1,27 @@
+# test-boot1-compatibility
+
+Runs all test under SpringBoot 1.x (Spring 4.x) environment.
+
+See build.gradle for details.
+
+## How to use line-bot-spring-boot under SpringBoot 1.x (Spring 4.x)
+
+In case of you want to use line-bot-spring-boot under SpringBoot 1.x,
+you should enforcing downgrade.
```

Comment:
If you want to use `line-bot-spring-boot` under SpringBoot 1.x, since `line-bot-spring-boot` depends on SpringBoot 2.x by default, you need to enforce to downgrade all SpringBoot libraries to 1.x (and Spring libraries from 5.x to 4.x).

Reply:
Thank you. Updated